zenoh-flow/tests/dataflow.rs

```diff
@@ -105,8 +107,8 @@ impl Node for GenericSink {
 // OPERATORS
 
 struct NoOp {
```

Comment:
I would add an Operator with `Input` and `RawOutput` (and vice-versa) as well as a `Sink` with `RawInput` just to verify that all cases work.